review.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTPYE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>
+          Dr. Norman Borlaug
+        </h1>
+        <h4>
+          The man who saved a billion lives
+        </h4>
+      </div>
+
+      <div id="img-div"> 
+        <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" >
+        <figcaption id="img-caption"> 
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+        </figcaption>
+      </div>
+    
+      <div id="tribute-info">
+        <p>
+          Here's a time line of Dr. Borlaug's life:
+        </p>
+      </div>
+
+      <div>
+        <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank" rel="noopener"></a>
```

Comment:
You need to add a placeholder for the hyperlink so that the user can click on it. 💯